plugins/janus_sip.c

```diff
@@ -3240,7 +3241,9 @@ static void *janus_sip_handler(void *data) {
 			session->media.ready = FALSE;
 			session->media.on_hold = FALSE;
 			janus_sip_call_update_status(session, janus_sip_call_status_closing);
-			nua_bye(session->stack->s_nh_i, TAG_END());
+			char custom_headers[2048];
+			janus_sip_parse_custom_headers(root, (char *)&custom_headers);
+			nua_bye(session->stack->s_nh_i, TAG_IF(strlen(custom_headers) > 0, SIPTAG_HEADER_STR(custom_headers)), TAG_END());
```

Comment:
Line a bit too long, better to split in multiple lines for readability (see `nua_invite` lines for instance).